### PR TITLE
test:add test_remove_snap_ImageNotFound test case in remove snap part

### PR DESCRIPTION
**Pull request by weiyingze-git**
Signed-off-by: Yingze Wei <weiyingze@cmss.chinamobile.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: src/test/pybind/test_rbd.py [hunk 1]
```diff
@@ -762,6 +762,14 @@ def test_remove_snap(self):
         self.image.remove_snap('snap1')
         eq([], list(self.image.list_snaps()))
 
+    def test_remove_snap_ImageNotFound(self):
```
Comment (dillaman) [inline src/test/pybind/test_rbd.py]:
```suggestion
    def test_remove_snap_not_found(self):
```

File: src/test/pybind/test_rbd.py [hunk 2]
```diff
@@ -762,6 +762,14 @@ def test_remove_snap(self):
...
+        eq([], list(self.image.list_snaps()))
```
Comment (dillaman) [inline src/test/pybind/test_rbd.py]: Nit: no need to retest listing/creating/removing snapshots -- just test remove against a non-existent snap

Comment (weiyingze-git) [reply src/test/pybind/test_rbd.py]: thank you vevry much 
i have updated

**Comment by zy751713126** [comment]
"test case" instead of "interface" is better in title, thanks.

**Pull request merged**